packages/components/atoms/f-form-field/src/components/FormDropdown.vue

```diff
@@ -7,7 +7,10 @@
             :data-test-id="testId.icon" />
         <select
             :id="$attrs.id"
-            :class="$style['c-formDropdown-select']"
+            :class="[
+                $style['c-formDropdown-select'],
+                { [$style['c-formDropdown--disabled']]: isDisabled }
```

Comment:
Rather than a class on the dropdown, I'd apply it as a disabled attribute (as that is valid).

So `:disabled="isDisabled"` should work, and then just update the CSS to use the disabled attribute.

You likely won't need the prop either, as the `attrs` are passed through already, so you can just make a computed like you do on the `formField` component that looks at the `$attrs.disabled` value